dev/fattest.simplicity/src/componenttest/topology/utils/MvnUtils.java

```diff
@@ -0,0 +1,305 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package componenttest.topology.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.ibm.websphere.simplicity.PortType;
+
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * A set of mostly static utility functions.
+ * This exists partly as there is more than one test class
+ * in development that share these functions.
+ */
+public class MvnUtils {
+
+    public static File home;
+    public static String wlp;
+    public static File tckRunnerDir;
+    public static boolean init;
+    public static String[] mvnCliRaw;
+    public static String[] mvnCliRoot;
+    public static String[] mvnCliTckRoot;
+    static List<String> jarsFromWlp = new ArrayList<String>(3);
+
+    /**
+     * Initialise shared values for a particular server.
+     * This enables us to set up things like the Liberty install
+     * directory and jar locations once.
+     *
+     * @param server
+     * @throws Exception
+     */
+    public static void init(LibertyServer server) throws Exception {
+        wlp = server.getInstallRoot();
+        home = new File(System.getProperty("user.dir"));
+        tckRunnerDir = new File("publish/tckRunner");
+
+        jarsFromWlp.add("com.ibm.websphere.org.eclipse.microprofile.config");
+        jarsFromWlp.add("com.ibm.ws.microprofile.config.cdi");
+        jarsFromWlp.add("com.ibm.ws.microprofile.config");
```

Comment:
As discussed, having a list of jars here in the common code isn't quite right.

Reply:
Yes. Once the list begins to grow I will generate it to be the set of ${variables} used in the tckRunner pom.xml. Otherwise - users need a 'list' of what ${variables} will work.

Reply:
I have removed this list by scanning the systemPaths used in the pom.xml using XPath.